ocpp/v16/call.py

```diff
@@ -133,7 +134,8 @@ class InstallCertificate:
 class RemoteStartTransaction:
     id_tag: str
     connector_id: Optional[int] = None
-    charging_profile: Optional[Dict] = None
+    # Accept Dict for backwards compatibility
+    charging_profile: Union[ChargingProfile, Dict, None] = None
```

Comment:
if this is left as optional, don't need to declare `None` as a type:

`charging_profile: Optional[Union[Dict, ChargingProfile]] = None`